.platform.app.yaml

```diff
@@ -147,12 +147,8 @@ web:
         '^/sites/default/files/(css|js)':
           expires: 2w
 
-# Set up migration content refresh every night with 30 minute gap between
-# file sync and db restoration on this app.
+# Set up migration content refresh every night.
 crons:
-  d7files:
-    spec: '30 0 * * *'
-    cmd: 'migrate-scripts/sync-files.sh $D7_PROJECT_ID $D7_PROJECT_ENV'
   d7db:
     spec: '0 1 * * *'
```

Comment:
The db fetch/unpack process takes between 20-30 minutes. This is set to run once a day, but clearly we can do it more often if we need to in future.